pegasus/sites.v3/hourofcode.com/i18n/pl.yml

```
@@ -2542,9 +2542,7 @@ pl:
   tutorial_allcancodeapps_shortdescription: Use the Allcancode Platform to create
     web and mobile apps from design to delivery.
   tutorial_n2y_shortdescription: Help Bitt Bott explore Earth using direction arrows.
-  tutorial_dance2019_longdescription: Code a Dance Party to share with your friends.
-    Featuring Katy Perry, Shawn Mendes, Lil Nas X, Panic! At The Disco, Jonas Brothers,
-    and many more!
+  tutorial_dance2019_longdescription: 'Koduj Potańcówkę, aby podzielić się ze znajomymi. Występują: Katy Perry, Shawn Mendes, Panic! At The Disco, Lil Nas X, Jonas Brothers, Nicki Minaj i 34 innych!'
```

I think this needs to be either
```suggestion
  tutorial_dance2019_longdescription: Koduj Potańcówkę, aby podzielić się ze znajomymi. Występują: Katy Perry, Shawn Mendes, Panic! At The Disco, Lil Nas X, Jonas Brothers, Nicki Minaj i 34 innych!
```

or 
```suggestion
  tutorial_dance2019_longdescription: "Koduj Potańcówkę, aby podzielić się ze znajomymi. Występują: Katy Perry, Shawn Mendes, Panic! At The Disco, Lil Nas X, Jonas Brothers, Nicki Minaj i 34 innych!"
```

---

I think the quotes are necessary? I don't have a preference for single or double quotes though

---

Hah, quite possibly. It's yaml, so who knows